backend/src/api/base.clj

```diff
@@ -13,13 +13,16 @@
   (player-view/get-game-as-player (persistence/fetch-game game-id) player-id))
 
 (defn play-card-as-player
-  [game-id player index row-id & target]
-  (let [game-state (persistence/fetch-game game-id)]
+ ([game-id player index row-id]
+  (play-card-as-player game-id player index row-id "nil"))
+ ([game-id player index row-id target]
+  (let [game-state (persistence/fetch-game game-id)
+        target (load-string target)]
```

Comment:
`load-string` is a security issue, it is executing arbitrary Clojure code contained in a string received from an external connection.
A malicious user could alter the value being sent as `target` and run any kind of code on the system, compromising the entire backend.

Reply:
Just as a reminder: this pull request *cannot* be accepted until it doesn't use vulnerable things such as `load-string`